tee-worker/core-primitives/types/Cargo.toml

```diff
@@ -29,6 +29,7 @@ sp-std = { default-features = false, git = "https://github.com/paritytech/substr
 
 # litentry
 litentry-primitives = { path = "../../litentry/primitives", default-features = false }
+parentchain-primitives = { package = "core-primitives", path = "../../../primitives/core", default-features = false }
```

Comment:
I think this can be chaotic because we have a crate:
=> depends on `parentchain-primitives` directly
=> depends on `litentry-primitives` that depend on `parentchain-primitives`

Can we make it linearly dependent? Like external crates only depend on `litentry-primitives`. `litentry-primitives` will re-export anything that it needs from `parentchain-primitives`

There're more places than this one